### PR TITLE
Update Zone lists again.

### PR DESCRIPTION
**Pull request by TeoTwawki**
@starlightknight & @gedads : just so you are aware mass extractor will already dump the mobs and NPCs in z288 (Escha- Zi'Tah) but I'll be making a  pull request to polutils soon (unless someone beats me to it) for the polutils menu's and to take care of mass extractor failing to fetch any dialog tables for zones > 255.

### CODE REVIEW

File: id-update-batch-jobs/binaries/config.properties [hunk 1]
```diff
@@ -566,14 +574,22 @@ mob-list-285.xml=Ferretory
...
+#279=scripts/zones/
 280=scripts/zones/Mog_Garden/
 281=scripts/zones/Leafallia/
 282=scripts/zones/Mount_Kamihr/
 283=scripts/zones/Silver_Knife/
 284=scripts/zones/Celennia_Memorial_Library/
 285=scripts/zones/Feretory/
+#286=scripts/zones/
+#287=scripts/zones/
+285=scripts/zones/Escha-ZiTah
```
Comment (xdemolish) [inline id-update-batch-jobs/binaries/config.properties]: missing /

File: id-update-batch-jobs/binaries/config.properties [hunk 2]
```diff
@@ -566,14 +574,22 @@ mob-list-285.xml=Ferretory
...
+#279=scripts/zones/
 280=scripts/zones/Mog_Garden/
 281=scripts/zones/Leafallia/
 282=scripts/zones/Mount_Kamihr/
 283=scripts/zones/Silver_Knife/
 284=scripts/zones/Celennia_Memorial_Library/
 285=scripts/zones/Feretory/
+#286=scripts/zones/
+#287=scripts/zones/
+285=scripts/zones/Escha-ZiTah/
```
Comment (starlightknight) [inline id-update-batch-jobs/binaries/config.properties]: You have two zone 285's

**Comment by TeoTwawki** [comment]
saw already fixing, also I forgot to bump max zone

**Comment by TeoTwawki** [comment]
https://github.com/DarkstarProject/darkstar-utils/pull/10 fixed

**Pull request merged**